.github/dependabot.yml

```diff
@@ -5,3 +5,7 @@ updates:
   schedule:
     interval: daily
   open-pull-requests-limit: 10
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: daily
```

Comment:
@jjbustamante we could add something like this to pack to ensure actions are kept up-to-date there